### ISSUE TITLE
Sugestions

### ISSUE THREAD

**Issue by lauren7ino** [dani7959/es-theme-replica#1]
Hi,
First of all, thank you for your great theme.
Sorry by basic English, this is not my mother language.

I'm using it in a RG353M in ArkOS and it's very nice!
I love gray on yellow mix.

I started playing with the fonts, to live up to the ZX Spectrum theme and the general feeling of those times.

I tested the War Games movie font in the main menu and it's beautiful.
This => https://fontmeme.com/fonts/lazenby-computer-font/
![IMG_0417](https://github.com/dani7959/es-theme-replica/assets/9194198/7b743e5c-8a44-489e-862c-a026492c1d75)

When I have some spare time, I will try to insert a ZX Spectrum font in the settings menu:
This => https://fontmeme.com/fonts/zx-spectrum-7-font/

As for the games list, it requires a compact font that I still didn't found.
The other two fonts are not adequate for this purpose.
War Games font is only readable in large fonts, and ZX Spectrum font is too large horizontally.
Maybe I'll try this => https://fontmeme.com/fonts/long-pixel-7-font/

**Comment by dani7959** [comment]
Hi, both Lazenby and ZX Spectrum fonts are great!
This is what I initially had in mind, but couldn't find them.

I will consider both, but first I need to do some calculations for all aspect ratios. This carousel font is also used in some logo words, such as "2003" from "MAME 2003" and "hacks" from "SNES hacks", so it's going to take a little bit of work.

As for the games list, I don't think I will theme the font, because the the battery and clock fonts cannot be themed and I like consistency.

Thanks a lot for your suggestions and enjoy :)

**Comment by dani7959** [comment]
Yeah, Lazenby is really great. Space efficient also. New carousel default!
Make sure you get the updated MAME 2003 and SNES HACKS logos ;-)

**Comment by lauren7ino** [comment]
Just tested your new version and it's great!
I was not sure it I was to update via ThemeMaster, so I uploaded the files by hand.
Is the ThemeMaster updated automatically when you update git?

**Comment by lauren7ino** [comment]
My MAME 2003 games were in a path called Arcade, so once I renamed the directory the logo appeared.
Very nice! Thanks!

**Comment by dani7959** [comment]
> Just tested your new version and it's great! I was not sure it I was to update via ThemeMaster, so I uploaded the files by hand. Is the ThemeMaster updated automatically when you update git?

I actually don't know if you can update via ThemeMaster. I know the guys added the theme to the github gallery, but I'm not sure about ThemeMaster.
And I don't want to insist and push my theme ;-)

**Comment by lauren7ino** [comment]
Found another beautiful font. It's a mix of old style "computer font" with a new twist:
https://all-free-download.com/font/download/computer_7_6905527.html

**Comment by lauren7ino** [comment]
On the lists I opted for this font:
https://www.dafont.com/apollo-2.font

**Comment by lauren7ino** [comment]
Please try my tweaks and give me your opinion.
Files attached.
Unfortunately I don't have much time to dedicate to tweaking.
Thanks

[ModifiedFiles.zip](https://github.com/dani7959/es-theme-replica/files/14705787/ModifiedFiles.zip)

**Comment by lauren7ino** [comment]
![image](https://github.com/dani7959/es-theme-replica/assets/9194198/c20898a9-4830-40ef-a2ee-bcb0fd1db546)
![image](https://github.com/dani7959/es-theme-replica/assets/9194198/a9f46f9d-4baf-4276-99e1-6280bfc931c7)

**Comment by dani7959** [comment]
Looks very good. I'll leave this discussion open, so that anyone can see and download your work.
For now, at least the header/footer will remain on the default font, because the battery indicator font cannot be changed.

One more thing ;-)
Backup your /roms/launchimages/loading.jpg and copy over the attached file.
It's using the ZX Spectrum font that you found!
![loading](https://github.com/dani7959/es-theme-replica/assets/35736735/ec1d1ebc-0874-4a11-97f6-89691a5f8083)

**Comment by lauren7ino** [comment]
Thanks! I have to try this.
I've bem using the text loading screens, but I believe it's time for a change.

As for Mame 2003, I had to revert to the original folder name "Arcade" because ES wound not let me swap emulators.
I will miss your logo.

Thanks, keep up the good work.

**Comment by dani7959** [comment]
There are separate logos for "arcade", "mame", and "mame 2003" folders.
"mame 2003" is a bit different because the font for the "2003" part is embedded by myself in the logo.
But you shouldn't have troubles with any of the included logos.
Every folder in the es_systems.cfg is themed except for retropie (options), vhs (movies) and ports.

Cheers, ;-)

**Comment by lauren7ino** [comment]
Hi
I'm using your loading logo and it's very nice. Thank you.
Although I usually use the brightness at 100% and I found the image too bright and flashy as I use the console while commuting, so I lowered the JPG text brightness to #d8d8d8 white, according to this site palette: https://lospec.com/palette-list/zx-spectrum
Thanks, keep up the good work

**Comment by lauren7ino** [comment]
Another loading ideia:

![image](https://github.com/dani7959/es-theme-replica/assets/9194198/27547dfa-9d77-40f9-9750-85bb85ed2704)

**Comment by lauren7ino** [comment]
Just for you to know, when I asked Theme Master to update the installed themes, it said it had an update with the text "Typo corrected".
This is the same text you have on your commit. So yes, this repo is synchronized with Theme Master.

**Comment by lauren7ino** [comment]
![loading](https://github.com/dani7959/es-theme-replica/assets/9194198/d34c1873-d8f1-4052-9139-52b12e8d73ac)

I just made this a loading image template.
Spectrum screen is 256x192, at 2x becomes 512x384, which gives us space to include the borders in our 640x480 image.

PS: I tried to add a LOADING ""L, without success as the font I doesn't render pixel perfect 8x8/16x16 chars.

**Comment by dani7959** [comment]
> Hi I'm using your loading logo and it's very nice. Thank you. Although I usually use the brightness at 100% and I found the image too bright and flashy as I use the console while commuting, so I lowered the JPG text brightness to #d8d8d8 white, according to this site palette: https://lospec.com/palette-list/zx-spectrum Thanks, keep up the good work

The light color used throughout the theme is c6c7c6
I did a lot of testing trying to have good contrast and legibility at all brightness levels.
But it looks like the contrast on black background is too much indeed.

Thank you for sharing ;-)
